mllib/src/main/scala/org/apache/spark/ml/pmml/PMMLExportable.scala

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
I think that might be good, the main difference is this avoids using the factory implementation that the MLLib API was.